### DISCUSSION TITLE
Drum Sampler Crashing

### DISCUSSION THREAD

**Discussion by kdr24912** [FundamentalFrequency/.github#56] [Q&A] [answered]
Hi all,
I finished my LMN build last night and it has been a blast to play with (I plan on posting pictures later on) but I ran into an issue with one of the plugins. When I select the 'DrumSampler' instrument my DAW program crashes to the Pi's desktop and I have to manually relaunch.

I've been able to use the 'Sampler' instrument and the '40SC' with no issues, it's only the drums crashing. I've tried loading multiple drum kits from briwil's post [here](https://github.com/orgs/FundamentalFrequency/discussions/51) with no success. 

My Pi 4 is running the v0.1.1 aarch64 version of the LMN DAW and I'll paste in the error code that is printed to the terminal when I select 'DrumSampler'.

Error:

> terminate called after throwing an instance of 'YAML::ParserException'
>   what():  yaml-cpp: error at line 1, column 8: unknown escape character: ?
> Aborted

Instead of driving myself insane trying to troubleshoot this error, I decided to post here in case somebody has also ran into this issue. If any further logs or info is needed, let me know. Any help you guys could give me would be much appreciated.

Thanks!

**Comment by benpocalypse** [comment, marked as answer]
Without being able to see all the Yaml files you have in your drumkits, I would wager that it looks like one of them is malformed with a "?" character in there according to the error message. Any chance you can open each of them manually to see if that's the case?
An easier approach might be to just remove all the drumkits you've added except for 1, and see if it works. If it does, add them back 1 by 1 and test each time until you find the culprit.

**Reply by kdr24912** [reply]
You were correct! I skimmed through the yaml in each kit and found one file with a question mark on it. Maybe I accidentally entered it in, who knows. But it's working properly now.
Thank you for the help. Have a good day.
